src/Region/CrtfImportExport.cc

```diff
@@ -417,10 +417,7 @@ RegionState CrtfImportExport::ImportAnnSymbol(casacore::CountedPtr<const casa::A
 
         // Set control points
         std::vector<CARTA::Point> control_points;
-        CARTA::Point point;
-        point.set_x(pixel_coords[0]);
-        point.set_y(pixel_coords[1]);
-        control_points.push_back(point);
+        control_points.push_back(Message::Point(pixel_coords[0], pixel_coords[1]));
```

Comment:
Given how frequently we construct a `Message::Point` from a `std::vector<casacore::Quantity>` or a `casacore::Vector<casacore::Double>`, I wonder if it would be worthwhile to add overloaded `Point` functions for these use cases (which take one of these vectors as a parameter, and have optional parameters for the x and y indices, which default to 0 and 1). Then we could e.g. write this as just `Message::Point(pixel_coords)`.

Reply:
I like that idea. Like this?
```c++
Message::Point(const std::vector<casacore::Quantiy>& input, int x_index = 0, int y_index = 1);
```

Reply:
Yup. Just for these two vector types; I didn't see any other cases that I think warrant additional simplification.